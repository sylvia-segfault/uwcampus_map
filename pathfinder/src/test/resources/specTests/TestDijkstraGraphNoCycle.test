# Find a path with Dijkstra's algorithm on
# a big graph that contains no cycle

CreateGraph graph
AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E
AddNode graph F
AddNode graph G
AddNode graph H
AddNode graph I
AddEdge graph C A 12.0
AddEdge graph C B 2.0
AddEdge graph C D 3.0
AddEdge graph C F 1.0
AddEdge graph E C 4.0
AddEdge graph D G 5.0
AddEdge graph G I 7.0
AddEdge graph G H 10.0
FindPath graph E I