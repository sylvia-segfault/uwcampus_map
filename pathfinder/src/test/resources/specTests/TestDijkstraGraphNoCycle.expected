# Find a path with Dijkstra's algorithm on
# a big graph that contains no cycle

created graph graph
added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph
added node F to graph
added node G to graph
added node H to graph
added node I to graph
added edge 12.000 from C to A in graph
added edge 2.000 from C to B in graph
added edge 3.000 from C to D in graph
added edge 1.000 from C to F in graph
added edge 4.000 from E to C in graph
added edge 5.000 from D to G in graph
added edge 7.000 from G to I in graph
added edge 10.000 from G to H in graph
path from E to I:
E to C with weight 4.000
C to D with weight 3.000
D to G with weight 5.000
G to I with weight 7.000
total cost: 19.000